src/styles/blocks/catalog.scss

```diff
@@ -17,9 +18,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: #{$transition};
```

Comment:
specify for which props you’re applying transition

Reply:
added